lib/smack/RustFixes.cpp

```diff
@@ -18,6 +18,17 @@ namespace smack {
 
 using namespace llvm;
 
+bool isRustNameMatch(StringRef search, StringRef name) {
```

Comment:
Actually this can be written as `return a || b`.

Reply:
I thought it would be less readable, but I think this is an improvement.